include/aws/common/logger.h

```diff
@@ -0,0 +1,102 @@
+#ifndef AWS_COMMON_LOGGER_H
+#define AWS_COMMON_LOGGER_H
+/*
+* Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License").
+* You may not use this file except in compliance with the License.
+* A copy of the License is located at
+*
+*  http://aws.amazon.com/apache2.0
+*
+* or in the "license" file accompanying this file. This file is distributed
+* on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+* express or implied. See the License for the specific language governing
+* permissions and limitations under the License.
+*/
+
+enum aws_log_level
+{
+    aws_off = 0,
```

Comment:
Should be something like AWS_LOG_LEVEL_OFF etc...

Reply:
OK